### ISSUE TITLE
Jenkins blog needs update

### ISSUE THREAD

**Issue by catlook** [rackerlabs/docs-developer-blog#220]
I have reached out to Trey Hoehne to find an appropriate resource.

Received following email from Yesenia Ceja <yesenia.ceja@RACKSPACE.COM>:

```I had a customer reach out to us with this article that needs some updating, the steps listed don’t match up with the current control panel structure. Do you know who I can reach out to for help with this? Author listed, Max Lincoln, is no longer at RAX.
https://developer.rackspace.com/blog/unlimited-jenkins-slaves-in-5-quick-steps/ ```

**Comment by catlook** [comment]
Ok, I had a conversation with Chris Hultin, and his recommendation was to delete this article, since it is out of date, and create a new article with the quick and easy command line equivalent.  I'll work with him to get that published!

**Comment by catlook** [comment]
closed with pr #226
